hadoop-ozone/dist/src/main/k8s/definitions/ozone/om-ss.yaml

```diff
@@ -36,6 +36,8 @@ spec:
         prometheus.io/port: "9874"
         prometheus.io/path: "/prom"
     spec:
+      securityContext:
+        fsGroup: 1000
       initContainers:
       - name: init
         image: elek/ozone
```

Comment:
I know, not part of this patch. But I hope we have a plan/JIRA to make this point to Ozone dockerhub. Thx